Jenkinsfile

```diff
@@ -347,6 +349,25 @@ lock(resource: "build-${params.STREAM}") {
             return
         }
 
+        // Do an Early Archive of just the OSTree. This has the
+        // desired side effect of reserving our build ID before
+        // we fork off multi-arch builds.
+        stage('Archive OSTree') {
+            if (s3_stream_dir) {
+              def force = params.FORCE ? "--force" : ""
```

Comment:
Hmm, I think here instead we want to always `--force` the upload. This covers the corner-case of the pipeline dying after/during `buildupload` but before updating `builds.json`. Gating this on `params.FORCE` seems odd. That parameter is to force the creation of a new build, not overwrite an existing one.

The other possibility is to swap the order and bump `builds.json` first, but then the risk is that we have a build ID for which we could in fact have 0 artifacts instead of at least having a `meta.json` and an OSTree. So I don't think that makes sense.

Reply:
added --force